Cargo.toml

```diff
@@ -34,7 +34,10 @@ libc = { version = "0.2.64", default-features = false }
 wasi = "0.9"
 
 [features]
+default = ["cpu"]
```

Comment:
Why enable it by default? De-facto it will be almost always enabled, since many crates will not use `default-featuers = false`, thus making the feature pointless. So I think we wither should make it a disabled by default feature, or do not feature-gate CPU-based implementation in the first place.

Reply:
I agree with this reasoning, and I think we should have the feature disabled by default. If a target is unsupported, we should not build it by default.

Reply:
Fixed, the `cpu` feature is now off by default.